<html><head></head><body><a href='javascript:history.back()'>Back</a> <a href='../index.html'>Top</a><dt><h3>Fail</h3></dt><dd><p align='justify'>Fail tag allows us to throw a failure at any point in the test 
               case where we feel that by reaching this point in the test we 
               have in fact hit an issue that should be noted as a failure of 
               the test case.</p></dd><br/><br/><dt><b>Author </b></dt><dd>Rodney Gomes</dd><br/><br/><dt><b>Optional Attributes</b></dt><dd><br/><table border=1><tr valign='top'><td width='100px'><b>message</b></td><td><p align='justify'>The message to be used when throwing the FailException.</p></td></tr></table></dd><br/><dt><b>Usage Examples</b></dt><br/><dd><b>Example #1</b></dd><dd><pre>&lt;?xml version="1.0" encoding="UTF-8" standalone="no"?&gt;
&lt;fail message="There was failure."/&gt;
</pre></dd><br/><dd><b>Example #2</b></dd><dd><pre>&lt;?xml version="1.0" encoding="UTF-8" standalone="no"?&gt;
&lt;fail&gt;
     Failed to meet some limit in the test
 &lt;/fail&gt;
</pre></dd></body></html>